docs/mkdocs.yml

```diff
@@ -24,4 +26,8 @@ plugins:
       minify_html: true
 
 nav:
-  - Introduction: 'index.md'
+  - Introduction: "index.md"
+  - Requirements: "requirements.md"
```

Comment:
Would cool add requirements page in `index.md`, not?

Reply:
Ok, it is cool, too! Done!